test.js

```diff
@@ -107,10 +107,13 @@ test('allow string arguments in synchronous mode', t => {
 	t.is(stdout, 'foo\nbar');
 });
 
-test('forbid string arguments together with array arguments', t => {
-	t.throws(() => {
-		execa('node fixtures/echo foo bar', ['foo', 'bar']);
-	}, /Arguments cannot be inside/);
+test('allow commands with spaces and array arguments', async t => {
+	const {stdout} = await execa('./fixtures/command with space', ['foo', 'bar']);
+	t.is(stdout, 'foo\nbar');
+});
+
+test('forbid commands with spaces and no array arguments', async t => {
+	await t.throwsAsync(execa('./fixtures/command with space'));
```

Comment:
Yes I thought this might be problematic as well as soon as @GMartigny mentioned `C:\Program File\node\node.exe`...

If that's ok with you, I will update this PR with:
  - `execa(file, [args], [options)` (previous behavior, no `command` logic)
  - `execa.command(command, [options])`

Reply:
Sounds good 👍